packages/paste-core/components/chat-composer/src/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import {AutoLinkNode, $getRoot, $createParagraphNode, $createTextNode} from '@twilio-paste/lexical-library';
+
+export const renderInitialText = (text: string): void => {
+  const root = $getRoot();
```

Comment:
question: can you explain how this `$getRoot()` stuff is working or add a link to where it comes from in lexical

Reply:
Because you can't pass a ref to the content editable, this is the workaround. It is a function that returns a reference to the content editable. There's no docs for this function unfortunately 😢